.github/workflows/maven.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  push:
+  repository_dispatch:
+    types: network_store_updated
```

Comment:
Also
```
on:
  push:
  repository_dispatch:
    types:
      - single_line_diagram_updated
```
?

Reply:
Since network_store_updated event is broadcasted when new modifications are pushed on powsybl-network-store or when single_line_diagram_updated event is received (see powsybl-network-store CI workflow), is it necessary to specifically trigger a build when single_line_diagram_updated event is received ?

Reply:
Ah I understand ok yes it's correct